themes/salix/assets/css/main.css

```diff
@@ -15,7 +19,6 @@ body {
 	padding: 0;
 }
 
-
```

Comment:
nit for future reference: I'd try to keep unrelated changes out of small commits. Just went to try and cherry-pick this over something else to experiment and ended up with merge conflicts because one of these spaces confused git (I think). No big deal, just a good general principal to follow.